src/describe.c

```diff
@@ -366,7 +366,7 @@ static int find_unique_abbrev_size(
 	int *out,
 	git_repository *repo,
 	const git_oid *oid_in,
-	int abbreviated_size)
+	unsigned int abbreviated_size)
 {
 	size_t size = abbreviated_size;
```

Comment:
One might wonder whether this is the right thing to do. Previously, passing `-1` was underflowing to a huge `size`, essentially saying "Do not abbreviate but print all digits". But the only callsite `show_suffix` gets called with `git_describe_format_options.abbreviated_size`, which is an `unsigned int` itself.